packages-native/bottom-sheet/README.md

```diff
@@ -1,3 +1,88 @@
-# Bottom drawer
+## Bottom sheet
 
-Highly configurable widget to introduce bottom sheet behavior.
+The bottom sheet widget creates a set of options while blocking interaction with the rest of the screen or a draggable surface anchored to the bottom of the screen. There are two variations: Modal bottom sheet and Expanding bottom sheet.
+
+## Usage
+
+​
+
+### Adding Triggerer for Modal
+
+​The triggerer is only used for modal bottom sheet and you need to select a boolean attribute. When it turns the value to true, the modal will be triggered.
+​
+
+### Adding Items
+
+For the sake of simplicity, there are two modes available to add new items to the bottom sheet:
```

Comment:
There are two modes which let you add new items to the bottom sheet: basic mode and custom mode.

Reply:
Done